tests/wpt/web-platform-tests/css/cssom/nullable-owner.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<head>
+  <meta charset="utf-8">
+  <title>ownerNode null test</title>
+  <link rel="help" href="https://drafts.csswg.org/cssom/#dom-stylesheet-ownernode">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <style></style>
+</head>
+<body>
+  <script>
+    test(function() {
+      let sheet = document.querySelector("style").sheet;
+      assert_true(sheet.ownerNode instanceof HTMLStyleElement, "ownerNode should be an instance of HTMLStyleElement");
+      sheet.ownerNode.remove();
+      assert_equals(sheet.ownerNode, null, "ownerNode should be null after removing the styleSheet");
```

Comment:
We should also assert that the style element's sheet is null, which now would fail. Basically, any call to stylesheet_owner_cleanup needs to also cleanup the owner's stylesheet. Mind filing a followup issue to extend this test and fix that?

Reply:
Ok, I can do that. So is there any change that I need to make in this pull request?